cmake/CMakeLists.txt

```diff
@@ -126,12 +126,10 @@ if (MSVC)
     # so that MSVCR*.DLL is not required at runtime.
     # https://msdn.microsoft.com/en-us/library/2kzt1wy3.aspx
     # This is achieved by replacing msvc option /MD with /MT and /MDd with /MTd
-    # http://www.cmake.org/Wiki/CMake_FAQ#How_can_I_build_my_MSVC_application_with_a_static_runtime.3F
+    # https://gitlab.kitware.com/cmake/community/wikis/FAQ#how-can-i-build-my-msvc-application-with-a-static-runtime
     foreach(flag_var
         CMAKE_CXX_FLAGS CMAKE_CXX_FLAGS_DEBUG CMAKE_CXX_FLAGS_RELEASE
```

Comment:
Where is onnxruntime_MSVC_STATIC_RUNTIME used?

Reply:
Why did you remove a lot of things?

Reply:
I followed the example verbatim given in the above link. Here's the full list of libraries that become part of the DLL. Notice that the C runtime libs (ucrt) are also part of it. Hence the C* flags seemed redundant.

![image](https://user-images.githubusercontent.com/2732907/55456098-fc6f7c00-559a-11e9-9136-ccf2b1f74231.png)

Reply:
The C flags are for any C source files. Unless you are sure onnxruntime and its dependencies won't have any C source file, I think they are needed.

Reply:
It shouldn't hurt. I can add them back.